src/Controller/QuestionTemplateController.php

```diff
@@ -26,15 +28,22 @@ class QuestionTemplateController extends AbstractController
      */
     private $questionTemplateService;
 
+    /**
+     * @var RepositoryManager
+     */
+    private $repoManager;
+
     /**
      * UserController constructor.
      * @param RendererInterface $renderer
      * @param QuestionTemplateService $questionInstanceService
+     * @param RepositoryManager $repoManager
      */
-    public function __construct(RendererInterface $renderer, QuestionTemplateService $questionInstanceService)
+    public function __construct(RendererInterface $renderer, QuestionTemplateService $questionInstanceService, RepositoryManager $repoManager)
     {
         parent::__construct($renderer);
         $this->questionTemplateService = $questionInstanceService;
+        $this->repoManager = $repoManager;
     }
 
     /**
```

Comment:
Same feedback as the other two. If this Controller uses only one Repository(and even if not) consider injecting the appropriate repositories instead of the repoManager.